src/database/standalone-sqlite.ts

```
@@ -3887,6 +4139,26 @@ if (!isMainThread) {
           worker.saveVerificationPriority(args[0], args[1]);
           parentPort?.postMessage(null);
           break;
+        case 'getIndexCleanupCandidateIds':
+          const cleanupCandidates =
+            worker.getIndexCleanupCandidateIds(args[0]);
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)